bot.py

```diff
@@ -29,7 +29,10 @@
 
 
 class Bot:
-    """A minimal wrapper for the Slack API, this class also manages the database"""
+    """
+    A minimal wrapper for the Slack API,
+    this class also manages the database
+    """
```

Comment:
Why break this up if it fits on one line?

Reply:
From pep8:

>For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

Reply:
Nvm my pycharm shows a ruler at the wrong column